frontend/src/components/Product/ProductOverviewItem/ProductOverviewItem.tsx

```diff
@@ -0,0 +1,46 @@
+import { Text } from '@fun-eat/design-system';
+import styled from 'styled-components';
+
+import type { ProductRanking } from '@/types/ranking';
+
+interface ProductRankingItemProps {
+  productRanking: ProductRanking;
```

Comment:
```suggestion
  rank?: number;
  name: string;
  image: string;
```

아직 상품 랭킹 관련 API가 없어서 감이 오지 않지만 랭킹이 있을수도 없을 수도로 표현할 수 있을거 같아요. 그러면 현재 variant도 rank 유무로 판단할 수 있으니 아래 스타일도 rank 유무로 판단하는건 어떤가요?

Reply:
좋습니다 그렇게 하면 따로 variant를 나눌 필요가 없어지네요! 한번 말씀하신대로 수정해봤는데 확인해주세요~~